packages/web-api/package.json

```diff
@@ -49,7 +49,7 @@
     "@slack/types": "^2.0.0",
     "@types/is-stream": "^1.1.0",
     "@types/node": ">=12.0.0",
-    "axios": "0.21.1",
+    "axios": "^0.24.0",
     "eventemitter3": "^3.1.0",
     "form-data": "^2.5.0",
     "is-stream": "^1.1.0",
```

Comment:
If we will merge this PR in favour of #1366 , we may still want to include [some of the devDependency updates #1366 introduces](https://github.com/slackapi/node-slack-sdk/pull/1366/files#diff-c37f41fce6c9e7fe08ba94502179041088c29e04666e107c98265272f8391039R76) in this PR too.